src/store/lokapi.ts

```diff
@@ -93,10 +103,11 @@ export var moduleLokAPI = {
     },
     auth_success(state: any) {
       state.status = 'success'
-      state.userProfile = lokApiService.userProfile
```

Comment:
Yes, sorry, I think I try to code as what I consider good architecture practice, and I understand it can cause issue since I don't have all the edge cases of the code in mind.